### PR TITLE
Add doc and improve the status page

### PR DESCRIPTION
**Pull request by remicalixte**
### What does this PR do?

- [x] Display aggregator stats in a new "Aggregator" section, not the "DogStatsD" section
- [x] Display dogstatsd stats in the "DogStatsD" section
- [x] Fix "Forwarder" section on the gui
- [x] Detail check status
- [x] Docs on the figures displayed on the status

### Motivation

Questions where asked on how to interpret the status page of agent 6

We should write a doc explaining what is everything: what is a sample, a flush, Metrics Samples vs Total Sample, how metrics samples are reset at each run where dogstatsd is not ...

### CODE REVIEW

File: Dockerfiles/cluster-agent/dist/templates/forwarder.tmpl [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
-  {{$key}}: {{$value}}
+    {{$key}}: {{$value}}
   {{- end }}
   {{- if .Transactions.DroppedOnInput }}
 
-  Warning: the forwarder dropped transactions because all workers were busy.
-  You should review your network performance, and tune the forwarder_num_workers
-  and forwarder_timeout options.
+    Warning: the forwarder dropped transactions because all workers were busy.
+    You should review your network performance, and tune the forwarder_num_workers
```
Comment (derekwbrown) [inline Dockerfiles/cluster-agent/dist/templates/forwarder.tmpl]: This is potentially misleading to me.  "Workers" is used in multiple places (when I first read I thought of the collector workers for checks), and probably isn't meaningful to users.

IMO we should have a shorter, more generic message with a link to a complete KB/manual page?

File: docs/agent/status.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- Checks Metric Sample: Total number of metrics sent from the checks to the aggregator
+- Dogstatsd Metric Sample: Total number of metrics sent from the dogstatsd server to the aggregator
+- Event: Total number of events sent to the aggregator
+- Service Check: Total number of service checks sent to the agrregator
+
+
+## Logs Agent
+
+## DogStatsD
+
```
[the remark this answers is not in the document]
Comment (remicalixte) [reply docs/agent/status.md]: For the logs agent I don't know if we should do it ourselves. I could add things for dogstatsd now

[the remark this answers is not in the document]
Comment (remicalixte) [reply docs/agent/status.md]: @arbll dogstatsd section is done. Can I merge this now ?

**Review by derekwbrown — APPROVED** [review]
I reread; it looks good to me.

**Pull request merged**